### DISCUSSION TITLE
Breaking changes since 0.16?

### DISCUSSION THREAD

**Discussion by Faithfinder** [saleweaver/python-amazon-sp-api#1377] [Q&A] [answered]
Joined a new project using this API, but it's on 0.16. I'd like to upgrade, but there's no changelog. I tried looking at the diff between tags, and it looks fairly safe to upgrade, but it's a big diff, so, anything I should be aware of??

**Comment by saleweaver** [comment, marked as answer]
No breaking changes since ever 😄
They will have a bunch of credentials you don't need anymore, but even those are silently ignored.
